DOC-1939: TinyMCE 6.4.1 Community docs release (#2658)
tinymce/tinymce-docs: modules/ROOT/pages/changelog.adoc

NOTE: This is the {productname} Community version changelog. For information about the latest {cloudname} or {enterpriseversion} Release, see: xref:release-notes.adoc[{productname} Release Notes].

== 6.4.1 - 2023-03-29

=== Fixed
* The `fontsizeinput` increase and decrease size buttons now work on TinyMCE mobile.

> * The TinyMCE editor toolbar is now accessible for all editor widths; it no longer collapses into an inaccessible vertical line at any presented editor width.

Author: ```suggestion
* The TinyMCE editor toolbar is now accessible for all screen widths; it no longer collapses into an inaccessible vertical line at any presented screen width.
```

> * The TinyMCE editor toolbar is now accessible for all screen widths; it no longer collapses into an inaccessible vertical line at any presented screen width.

Reviewer: Just for absolute clarity

```suggestion
* The TinyMCE editor toolbar is now accessible for all screen widths; it no longer collapses into an inaccessible vertical line when the screen is scrolled horizontally.
```
